base/compiler/optimize.jl

```diff
@@ -400,17 +406,29 @@ function renumber_ir_elements!(body::Vector{Any}, changemap::Vector{Int})
 end
 
 function renumber_ir_elements!(body::Vector{Any}, ssachangemap::Vector{Int}, labelchangemap::Vector{Int})
+    delta = abs(labelchangemap[1])
```

Comment:
Some of the changes here and below are cosmetic (like the `i->j` renaming I made to make it easier to interpret `@show` while debugging), but I was a bit worried about the early-exit criterion in this function. Previously it assumed that the sum of all changes wasn't 0; that's valid if you only ever grow the number of lines, but since this might someday be used for deletions I thought it would be better to be a bit more careful.